src/wallet/wallet_rpc_server.cpp

```diff
@@ -31,6 +31,7 @@
 #include <boost/asio/ip/address.hpp>
 #include <boost/filesystem/operations.hpp>
 #include <boost/algorithm/string.hpp>
+#include <boost/optional/optional.hpp>
```

Comment:
This is unnecessary as `boost::optional` is already used in `wallet2.h`.

Reply:
Thanks. I missed that.